### PR TITLE
Saving the plots

### PR DESCRIPTION
**Pull request by amrukwa**
first commit is fix for the build, I will be adding the feature after that

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ## ROC'n'ROLL
 
-### Version 0.1.1
```
Comment (ArturOle) [inline changelog.md:3]: The historical logs should be left beneath the newest log (in this case 0.1.3). Pretty please, fill previously removed logs

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 ## Deployment
 To bundle into a single platform specyfic executable, run:
     
-    pyinstaller -F -w app.py
+    pyinstaller -F -w src/app.py
```
Comment (ArturOle) [inline README.md:11]: For now, it is ok, but there is compile.py which will be used as the python wrapper for this command

File: src/app.py [hunk 3]
```diff
@@ -1,5 +1,7 @@
 from math import sin
 
+import plotly.graph_objects as go
```
Comment (ArturOle) [inline src/app.py:3]: Please, consider chainging the go into something more meaningful. (Even something like graph_object)

**Review by ArturOle — CHANGES_REQUESTED** [review]
Please, consider the comments I have left you

File: src/app.py (hunk 3 above)
Comment (amrukwa) [reply src/app.py:3]: sure, but conventionally it is imported this way. I will change it for the project purposes, though.

**Review by ArturOle — APPROVED** [review]
Okay, everything looks right now. Game_ob is much more understandable in our case. If this would be a file where we mainly focus on these game_objects then the "go" convention would not be as problematic. In our case, it might be confusing (go? like where?). In the case of pandas as pd and NumPy as np there is no problem because it can't be confused with anything else. Thank you for these changes!

**Pull request merged**